documentation/content/en/books/handbook/introduction/_index.adoc

```diff
@@ -98,12 +98,14 @@ Here is just a sampling of some of the applications in which people are currentl
 ** Firewalls and NAT ("IP masquerading") gateways
 ** FTP servers
 ** Email servers
+** Storage servers
+** Virtualization servers
 ** And more...
 
 * _Education:_ Are you a student of computer science or a related engineering field? There is no better way of learning about operating systems, computer architecture and networking than the hands-on, under-the-hood experience that FreeBSD can provide. A number of freely available CAD, mathematical and graphic design packages also make it highly useful to those whose primary interest in a computer is to get _other_ work done!
 * _Research:_ With source code for the entire system available, FreeBSD is an excellent platform for research in operating systems as well as other branches of computer science. FreeBSD's freely available nature also makes it possible for remote groups to collaborate on ideas or shared development without having to worry about special licensing agreements or limitations on what may be discussed in open forums.
 * _Networking:_ Need a new router? A name server (DNS)? A firewall to keep people out of your internal network? FreeBSD can easily turn that unused PC sitting in the corner into an advanced router with sophisticated packet-filtering capabilities.
-* _Embedded:_ FreeBSD makes an excellent platform to build embedded systems upon. With support for the ARM(R), MIPS(R) and PowerPC(R) platforms, coupled with a robust network stack, cutting edge features, and the permissive extref:{faq}[BSD license, bsd-license-restrictions], FreeBSD makes an excellent foundation for building embedded routers, firewalls, and other devices.
+* _Embedded:_ FreeBSD makes an excellent platform to build embedded systems upon. With support for the ARM(R) and PowerPC(R) platforms, coupled with a robust network stack, cutting edge features, and the permissive extref:{faq}[BSD license, bsd-license-restrictions], FreeBSD makes an excellent foundation for building embedded routers, firewalls, and other devices.
```

Comment:
I'd include aarch64 here too. Maybe even risc-v too. Those two platforms are very active in the embedded space, though aarch64 is quite a bit more mature than the others except maybe arm.

Also, the (R) stuff is excessive, imho, but i'm not read in on the full context and practices here. elsewhere, we're quite sloppy about adding trademarks for other people's trade names since usually one needn't do that unless one is using those trade names to endorse the product...

Reply:
Agreed on all points. 

Are there precedences for "All trademarks are the property of their respective owners." ? I have removed the (R)'s short of UNIX(R) and added that phrase to the bottom.

As the page has received attention since I opened the PR, lessons will be learned on how this works in Git|GitHub.